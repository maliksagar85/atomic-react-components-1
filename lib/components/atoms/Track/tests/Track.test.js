import React from 'react';
import { shallow } from 'enzyme';

import { TrackVanilla } from '../index';

describe('<Track />', () => {
  let TrackComponent = '';
  beforeEach(() => {
    TrackComponent = shallow(<TrackVanilla>Test</TrackVanilla>);
  });

  test('should render correctly', () => {
    expect(TrackComponent).toMatchSnapshot();
  });
});
