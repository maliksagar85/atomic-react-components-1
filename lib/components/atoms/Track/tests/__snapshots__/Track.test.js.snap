// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Track /> should render correctly 1`] = `
<track
  kind="subtitles"
/>
`;
